.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pass
```

Comment:
Let's separate the 2 sections of the gitignore with comments like `# TaylorBot secret files` and `# macOS` :)